dist/formats/specialist_document/frontend/schema.json

```diff
@@ -1467,6 +1465,7 @@
               "montenegro",
               "montserrat",
               "morocco",
+              "north-macedonia",
```

Comment:
Think these need to go slightly further down between North Korea and North Sudan as you have for the health certificates

Reply:
Thanks for spotting @JonathanHallam 💯. I've remembered how the alphabet works now, and pushed up a fix.